rllib/tuned_examples/sac/atari-sac.yaml

```diff
@@ -31,23 +31,19 @@ atari-sac-tf-and-torch:
         n_step: 1
         rollout_fragment_length: 1
         replay_buffer_config:
-          _enable_replay_buffer_api: False
-          type: MultiAgentReplayBuffer
+          type: MultiAgentPrioritizedReplayBuffer
           capacity: 1000000
           # How many steps of the model to sample before learning starts.
-          learning_starts: 1500
+          learning_starts: 100000
```

Comment:
Is this a mistake? Why change this from 1.5k to 100k?

Reply:
There where two `learning_starts` in the config - so I had to pick one and chose the one that was there in our last release!